cimr/processor/lift.py

```diff
@@ -0,0 +1,84 @@
+"""Update genomic positions to hg38 reference.
+Uses pyliftover 0.4 currently, which only allows
+lifting over point coordinates and not indels, etc.
+"""
+
+
+__author__ = 'yoson park'
+
+import os
+import sys
+import pandas
+import logging
+import pyliftover
+
+from ..defaults import (HG19TO38, HG18TO38)
+
+
+def convert_coords(lifting, chrom, pos):
+    """Using pyliftover to convert genomic positions from
+    older reference to the GRCh38/hg38 reference.
+    """
+    _lifted_chrom = 'NA'
+    _lifted_pos = 'NA'
+    try:
+        pos = int(pos)
+        lifted = lifting.convert_coordinate(chrom, pos)
+        if lifted:
+            if len(lifted) > 1:
+                logging.warning(f' Liftover with more than one candidate: {t.variant_id}')
+            _lifted_chrom = lifted[0][0]
+            _lifted_pos = lifted[0][1]
+    except:
+        pass
```

Comment:
Not sure whether all exceptions here should be swallowed silently. For example, if `pos` is not a valid integer, should we at least log it as a warning?

Reply:
That is a good point. I will add a debugging log here. Potentially it could spit out a lot of logs as this is done line by line of the input file.